pom.xml

```diff
@@ -0,0 +1,645 @@
+                                    <value>${project.build.directory}/allure-results</value>
+                                </property>
+                            </systemProperties>
+                        </configuration>
+                        <dependencies>
+                            <dependency>
+                                <groupId>org.aspectj</groupId>
+                                <artifactId>aspectjweaver</artifactId>
+                                <version>${aspectj.version}</version>
+                            </dependency>
+                        </dependencies>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>on-prem-suite</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>${maven-surefire-plugin.version}</version>
+                        <configuration>
+                            <systemProperties>
+                                <testFailureIgnore>false</testFailureIgnore>
+                                <argLine>
+                                    -Dfile.encoding=UTF-8
+                                    -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
+                                </argLine>
+                                <property>
+                                    <name>env</name>
+                                    <value>on-prem</value>
+                                </property>
+                                <property>
+                                    <name>cucumber.publish.quiet</name>
+                                    <value>true</value>
+                                </property>
+                                <property>
+                                    <name>cucumber.plugin</name>
+                                    <value>html:target/cucumber-html-reports.html,io.qameta.allure.cucumber6jvm.AllureCucumber6Jvm</value>
+                                </property>
+                                <property>
+                                    <name>allure.results.directory</name>
+                                    <value>${project.build.directory}/allure-results</value>
+                                </property>
+                            </systemProperties>
+                        </configuration>
+                        <dependencies>
+                            <dependency>
+                                <groupId>org.aspectj</groupId>
+                                <artifactId>aspectjweaver</artifactId>
+                                <version>${aspectj.version}</version>
+                            </dependency>
+                        </dependencies>
+                    </plugin>
```

Comment:
Can you add this to a pluginManagement section and just reference the plugin in this section so that we don't need to repeat the configuration everytime?

Reply:
added plugin management for plugin dependencies.
System properties overwrite the global configuration set by plugin management.

Reply:
this has been done and verified